libs/datasets/sources/covid_tracking_source.py

```diff
@@ -111,9 +111,7 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         data[cls.Fields.COUNTY] = None
         data[cls.Fields.COUNTRY] = "USA"
         data[cls.Fields.AGGREGATE_LEVEL] = AggregationLevel.STATE.value
-        # Date checked is the time that the data is actually updated.
-        # assigning the date field as the date floor of that day.
-        data[cls.Fields.DATE] = data[cls.Fields.DATE_CHECKED].dt.tz_localize(None).dt.floor("D")
+        data[cls.Fields.DATE] = pd.to_datetime(data[cls.Fields.DATE], format="%Y%m%d")
```

Comment:
Could this be fixed by adding `cls.Fields.DATE` to `parse_dates`? or does it not recognize the date format

Reply:
Fixed in c126577